CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 23.4.0
 
+New error codes:
+* Y055: Unions of the form `type[X] | type[Y]` can be simplified to `type[X | Y]`.
+  Similarly, `Union[type[X], type[Y]]` can be simplified to `type[Union[X, Y]]`.
+
+Other changes:
```

Comment:
I released v23.4.0 earlier today, so this won't be in that version -- you should add a new `## Unreleased` section at the top, and put the changelog entry in that section :)